src/main/java/Main.java

```diff
@@ -0,0 +1,15 @@
+public class Main {
+    public static void main(String[] args) {
+
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#use-abstract-reference-instead-of-specific-one-where-possible): Use abstract reference instead of specific one where possible: